lib/library.go

```diff
@@ -61,6 +62,20 @@ func StopNode() *C.char {
 	return makeJSONResponse(nil)
 }
 
+// Create an X3DH bundle
+//export CreateX3DHBundle
+func CreateX3DHBundle() *C.char {
+	bundle, err := statusBackend.CreateX3DHBundle()
+	if err != nil {
+		return makeJSONResponse(err)
+	}
+
+	cstr := C.CString(bundle)
+
+	defer C.free(unsafe.Pointer(cstr))
```

Comment:
@adambabik I actually wanted to check, is not currently used this method, it will be used only to share the bundle with your contact code, this part needs to be looked up a bit.

Reply:
@adambabik This has been a recurring issue in many PRs. Is it something that'll get solved soon with gomobile? Otherwise, maybe we should do a PR that exposes a method to free memory.